### DISCUSSION TITLE
Using my own pore model as an input

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#11] [Q&A] [answered]
Hello,

I am trying to use my own pore model as an input for modeling the multiphasic flow. (Test Case 02).
When I put my raw image with 512*512 pixels in the raw folder of images, an error pops up and mentions that reshaping array is not possible.
Could you please help me out on how can I use the raw image of my pore model in Test Case 02?
What are the steps for inputting my pore model?

Looking forward to hearing from you.
Best,
Foojan

**Comment by GeoChemFoam** [comment, marked as answer]
The input images are not actually 2D. They are 2D slices repeated ten times in the z direction. You can duplicate your image 9x with fiji and then do images to Stack and it will create a 512x512x10 model which should work.

On Aug 8, 2023, at 06:49, Foojan-Ka ***@***.***> wrote:

﻿
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hello,

I am trying to use my own pore model as an input for modeling the multiphasic flow. (Test Case 02).
When I put my raw image with 512*512 pixels in the raw folder of images, an error pops up and mentions that reshaping array is not possible.
Could you please help me out on how can I use the raw image of my pore model in Test Case 02?
What are the steps for inputting my pore model?

Looking forward to hearing from you.
Best,
Foojan

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/11>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFQ2ZHWB4X24HIDL4P3XUHHQFANCNFSM6AAAAAA3H52GLM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Reply by foojan-Ka** [reply]
Thanks a lot for your help.
